Platform/CoffeelakeBoardPkg/BoardConfig.py

```diff
@@ -74,9 +74,8 @@ def __init__(self, *args, **kwargs):
         self.ENABLE_FWU           = 1
         self.ENABLE_SMBIOS        = 1
 
-        # FWU_PLD | PLD | Stage2 | Stage1B
         # Stage1B is verified by ACM
-        self.VERIFIED_BOOT_HASH_MASK  = 0x000000E
+        self.VERIFIED_BOOT_STAGE_1B  = 0x0

```

Comment:
Let us remove this since it is the same as default value 0.

Reply:
Done. Pushed another version.